src/util/clone.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @function
+ * @description Deep clone a class instance.
+ * @param instance The class instance you want to clone.
+ * @returns A new cloned instance.
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
i would like you to try declaring this like the function [here](https://github.com/oldschoolgg/oldschooljs/blob/b7ecc192613dece212023c724a54a51ecfe7180c/src/util/util.ts#L191) before disabling eslint.  Using a template type is more explicit too since you know whats going in would be whats coming out